protocol/deputy-guardian-module-improvements.md

```diff
@@ -0,0 +1,125 @@
+# Deputy Guardian Module Improvements
+
+## Context
+
+The Foundation Safe is given the ability to act as the Security Council Safe for a limited number
+of safety-net actions on OP Mainnet. For instance, the Foundation Safe can act as the Security
+Council Safe to trigger the Superchain-wide pause function. These powers are given to the
+Foundation Safe through the `DeputyGuardianModule` installed in the Security Council Safe.
+
+## Problem Statement
+
+The existing `DeputyGuardianModule` has a number of problems that we'd like to be able to solve in
+a simple manner that makes as few contract changes as possible. We describe each of these problems
+individually for clarity.
+
+### Pre-Signed Pause Transactions
+
+The Foundation Safe pre-signs certain transactions to reduce response time in case of an emergency.
+Because pre-signed Foundation Safe transactions are dependent on the nonces of that Safe, these
+pre-signed transactions become invalidated any time the Foundation Safe is required to sign
+literally any other transaction. In an attempt to reduce the overhead from this state of affairs,
+a *second* Foundation Safe sometimes referred to as the "Foundation Operations Safe" with the exact
+same configuration as the original Foundation Safe was made to be the Deputy Guardian.
+
+Even with this second Foundation Safe, pre-signed pauses are invalidated on a regular basis
+whenever an upgrade touches the `DeputyGuardianModule`. Gripes with the pre-signed pause system
+could fill a whole role of toilet paper and are not just limited to the issues noted above.
+
+### Unclear Responsibilities
+
+The `DeputyGuardianModule` has an unclear security model and underlying purpose. The original
+intent behind the `DeputyGuardianModule` was to provide a fast way to respond to potential
+incidents in a manner that can impact liveness but not system safety. A lack of strict clarity
+behind this design has meant that the `DeputyGuardianModule` has become a source of security
+implications that are not often easy to reason about. We would like to be more strict about what
+the `DeputyGuardianModule` can and cannot do.
+
+## Proposed Solution
+
+We propose resolving the above problems by modifying the existing `DeputyGuardianModule` to be more
+restrictive in the actions it can carry out while also being more permissive in who can carry out
+these actions.
+
+The `DeputyGuardianModule` would be permitted to:
+
+1. Trigger the Superchain-wide pause.
+1. Blacklist an individual dispute game.
+1. Change the respected game type to the `PermissionedDisputeGame`.
+1. Trigger the `setAnchorState` function in the `AnchorStateRegistry`.
```

Comment:
IMO allowing the Guardian to trigger the "fallback" to the permissioned game goes beyond liveness considerations and is what muddies the waters. Would prefer to have only the first two items in this list available.

Reply:
I generally agree with you, this was my original proposal but see conversation with Adrian above for why I changed it